packages/report-server/src/routes/FetchReportRoute.ts

```diff
@@ -33,6 +39,19 @@ export class FetchReportRoute extends Route<FetchReportRequest> {
     return report;
   }
 
+  private getHasData = (results: BuiltReport['results']) => {
+    // If the results are null or undefined, there is no data
+    if (!isNotNullish(results)) return false;
```

Comment:
Nitpicky, but this reads a little awkward—does it it warrant an `isNullish()`?

Reply:
Sure, have updated